TombEngine/Objects/TR2/Entity/tr2_skidman.cpp

```diff
@@ -310,8 +311,8 @@ namespace TEN::Entities::Creatures::TR2
 			if (skidooItem->RoomNumber != riderItem.RoomNumber)
 				ItemNewRoom(riderItemNumber, skidooItem->RoomNumber);
 
-			riderItem.Animation.AnimNumber = skidooItem->Animation.AnimNumber + (Objects[ID_SNOWMOBILE_DRIVER].animIndex - Objects[ID_SNOWMOBILE_GUN].animIndex);
-			riderItem.Animation.FrameNumber = skidooItem->Animation.FrameNumber + (GetAnimData(riderItem).frameBase - GetAnimData(skidooItem).frameBase);
+			riderItem.Animation.AnimNumber = skidooItem->Animation.AnimNumber;
```

Comment:
As with skateboard kid, I think what happens here is animation synchronization. Object needs to be tested.